### ISSUE TITLE
Can't specify config file for serverless components

### ISSUE THREAD

**Issue by kueckermann** [serverless/components#764]
# Description
In the serverless cli docs it specifies that by using **-c** you can specify an alternative deployment file.
If I try to specify a file I get `"service" property is missing in serverless.dev.yml`
Note that it will default to using serverless.yml if it is present in the current folder.

## Additional Data
I either need to use a separate file or be able to use serverless.js (https://github.com/serverless/components/issues/762), so that I can specify different options for my dev and prod environments. 

E.g. stack trace, version, operating system, ...
Operating System:          darwin
Node Version:              12.16.3
Framework Version:         1.78.1
Plugin Version:            3.7.0
SDK Version:               2.3.1
Components Version:        2.34.3

**Comment by eahefnawy** [comment]
yeah, this is only available in the Serverless Framework CLI, not the Components CLI.

**Comment by Enase** [comment]
@eahefnawy there are tons of the same issue reporting. Everyone expects to use variables in same same way as they use it in serverless framework cli. is there any plans to fix or change it?
